apps/openmw/mwgui/mapwindow.cpp

```diff
@@ -165,7 +165,11 @@ namespace MWGui
         , mCompass(NULL)
         , mChanged(true)
         , mFogOfWarToggled(true)
+#if defined (ANDROID) && !defined (OPENGL_ES)
+        , mFogOfWarEnabled(false)
```

Comment:
And you get warning `unused argument`. Can you move this code where it is called? Outside ctor?